crypto/x509util/profile.go

```diff
@@ -312,6 +335,27 @@ func (b *base) SetSubjectPublicKey(pub interface{}) {
 	b.subPub = pub
 }
 
+func (b *base) AddExtension(ext pkix.Extension) {
+	b.ext = append(b.ext, ext)
+}
+
+func (b *base) RemoveExtension(oid asn1.ObjectIdentifier) {
+	for i, ext := range b.ext {
+		if ext.Id.Equal(oid) {
+			b.ext = append(b.ext[:i], b.ext[i+1:]...)
+			break
+		}
```

Comment:
Any risk of there being multiple extensions with the same oid?

Reply:
The way is being used, no.